apps/client/src/app/conferences/conferences.module.ts

```diff
@@ -59,6 +60,7 @@ import { ClientSharedUiInputModule } from '@conf-match/client/shared/ui-input';
     ZXingScannerModule,
     ConferenceResolversModule,
     ClientSharedUiInputModule,
+    ContentLoaderModule,
```

Comment:
Are we actually making use of this package? It seems like a cool package, but if I turn it off locally I get the same experience where the "loading conferences" text shows and then it goes to the conference list. I'm wondering if we actually need to add this package?